lib/net/ldap/pdu.rb

```diff
@@ -84,7 +84,7 @@ class Error < RuntimeError; end
 
   def initialize(ber_object)
     begin
-      @message_id = ber_object[0].to_i
+      @message_id = ber_object[0]
```

Comment:
What's the benefit of removing this coercion?

Reply:
@ggalancs :point_up:

Reply:
@HarlemSquirrel Believe this was to prevent a few casting issues when the item could not be converted into an integer. Run into a few of these myself when testing.